src/plugins/lua/mime_types.lua

```diff
@@ -53,7 +53,7 @@ local settings = {
 
   bad_extensions = {
     cue = 2,
-    exe = 1,
+    exe = 2,
```

Comment:
IMO 2 is too much for default configuration as it will affect some legitimate messages.

Reply:
Thank you, I just reverted the offending commit.